rllib/algorithms/ddpg/ddpg.py

```diff
@@ -270,12 +270,6 @@ def validate(self) -> None:
                 f"Try setting config.rollouts(rollout_fragment_length={self.n_step})."
             )
 
-        if self.model["custom_model"]:
```

Comment:
apparently we never called this validate() function method before when custom_model was provided.